subsys/net/lib/lwm2m/lwm2m_registry.c

```diff
@@ -778,7 +778,11 @@ int lwm2m_set_s64(const struct lwm2m_obj_path *path, int64_t value)
 
 int lwm2m_set_bool(const struct lwm2m_obj_path *path, bool value)
 {
-	uint8_t temp = (value != 0 ? 1 : 0);
+	uint8_t temp = 0;
```

Comment:
Empty line is needed after variable definition.
Also, please add a short comment in the code why is this needed.